django_project/minisass_frontend/src/pages/Howto/index.tsx

```diff
@@ -1158,7 +1160,130 @@ const HowtoPage: React.FC = () => {
             src={`${newURL}img_rectangle6_280x303.png`}
             alt="rectangleSix_One"
           />
-          <MiniSASSResources />
```

Comment:
In future, these videos will need to be updated and the links changed. Could we have an admin menu that would show the following:
* Video Link
* Tittle etc so that an admin can update these videos / content